televisionbingecalculator/src/main/AndroidManifest.xml

```diff
@@ -1,54 +1,67 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.vanillax.televisionbingecalculator.app">
-
-	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-	<uses-permission android:name="android.permission.INTERNET" />
-
-	<supports-screens android:smallScreens="false">
-	</supports-screens>
-
-	<uses-permission android:name="android.permission.WAKE_LOCK" />
-	<application
-			android:name=".tbc.TelevisionBingeCalculator"
-			android:allowBackup="true"
-			android:hardwareAccelerated="true"
-			android:icon="@mipmap/ic_launcher"
-			android:label="@string/app_name"
-			android:theme="@style/AppTheme">
-		<activity
-				android:name=".kotlin.LandingActivity"
-				android:label="@string/app_name"
-				android:screenOrientation="portrait">
-			<intent-filter>
-				<action android:name="android.intent.action.MAIN" />
-
-				<category android:name="android.intent.category.LAUNCHER" />
-			</intent-filter>
-		</activity>
-		<activity
-				android:name=".kotlin.DetailsActivity"
-				android:label="@string/title_activity_show_details"
-				android:parentActivityName=".kotlin.LandingActivity"
-				android:screenOrientation="portrait">
-			<meta-data
-					android:name="android.support.PARENT_ACTIVITY"
-					android:value=".kotlin.LandingActivity" />
-		</activity>
-
-		<meta-data
-				android:name="com.crashlytics.ApiKey"
-				android:value="1db1f617dd035d3cf864f4501cfa7274cfba1808" />
-
-		<!--<activity-->
-				<!--android:name=".view.LandingActivityMain"-->
-				<!--android:windowSoftInputMode="adjustNothing">-->
-			<!--<intent-filter>-->
-				<!--<action android:name="com.google.android.gms.actions.SEARCH_ACTION" />-->
-
-				<!--<category android:name="android.intent.category.DEFAULT" />-->
-			<!--</intent-filter>-->
-		<!--</activity>-->
-	</application>
+<manifest
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.vanillax.televisionbingecalculator.app"
+    >
+
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <supports-screens android:smallScreens="false"></supports-screens>
+
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <application
+        android:name=".tbc.TelevisionBingeCalculator"
+        android:allowBackup="true"
+        android:hardwareAccelerated="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:theme="@style/AppTheme"
+        tools:ignore="GoogleAppIndexingWarning"
+        >
+        <activity
+            android:name=".kotlin.LandingActivity"
+            android:label="@string/app_name"
+            >
+        </activity>
+
+        <!--this will alow us in future to change to a different launcher activity without user loosing their homescreen shortcut-->
+        <activity-alias
```

Comment:
This is cool! I have never seen this before

Reply:
yeah right cool.

Reply:
Freebie that Mitch gets as part of deal.